### PR TITLE
Docs: Overhaul of Getting started guide

### LINKED ISSUE #6611: Getting started guide needs an overhaul
In #6587, @ktsaou brought up concerns about Netdata users not being made aware that if they want to store metrics for more than an hour (the default), they can either expand their `history` config or change the `memory mode` config to `dbengine`.

I reviewed the [getting started guide](https://docs.netdata.cloud/docs/gettingstarted/), and am now looking into a revision/overhaul of the guide to better address some of Netdata's new features and serve new users better. I'd like to hear from the community about what parts of this guide are good, what isn't good, what's not necessary or too complex for the true beginner users, and what else should be added.

Here's my revised table of contents, along with my thoughts on how I'd like to change the content. Feedback is welcome.

---

## Access the dashboard

Re-organize content and add more details about finding a system's IP address. Add basic information about navigating the interface.

## Change how long Netdata stores metrics

Change heading from **Sizing Netdata**. Retain content about changing the `history` config, but include more real-world numbers. Add details about changing `memory mode` to the new `dbengine` option, and link to existing DB engine documentation.

## Configuration basics

Change heading from **Configuration quick start**. Add information about the most important items in `netdata.conf`, such as `history`, `memory mode`, etc. (**I could use some thoughts on which configurations are most useful and most commonly changed.**)

Retain information about configuring plugins using `python.d` and `nginx` as an example, but simplify and clarify as needed. 

I also believe the content under the **Service discovery and auto-detection** heading could be condensed and added to this section. Basically, clarify that a ton of stuff is auto-detected, so they don't have to configure/enable every plugin.

## Health monitoring and alarms

Retain this information. Clarify and simplify if needed.

## Performance troubleshooting

We need beginner-level information about how to transform an alarm or anomalous metrics visible on one or more charts into actionable insights on the system. Perhaps a real-world example or two is in order. For example: "Your CPU is high. Go to the **Applications** section and look at the list of applications using the CPU for more information."

## Monitoring multiple systems with Netdata

Move the content beginning with "When you install multiple Netdata servers..." here. Add information about connecting nodes to Netdata Cloud.

## Start and stop Netdata

Move this to the end, since it seems less important to me. Emphasize that Netdata is configured by default to start at boot.

## What's next?

Retain existing links. Add link to Nginx proxy guide.

### PR DESCRIPTION
**Pull request by joelhans**
##### Summary

Fixes #6611 

The Getting started guide was in need of an overhaul, given how popular it is. This PR adds some new content, makes clearer the existing content, and removes some of the more obscure/niche things that most Netdata users won't be too interested in.

I've also moved the file from `docs/GettingStarted.md` to `docs/getting-started.md`, as the latter is a more standard naming structure for slugs. I've reflected that in the sidebar and I've tried my hand at creating a Netlify redirect from one to the other that I'll test once the test deploy finishes.

And, this PR comes with a tutorial that I've placed at `docs/tutorials/longer-metrics-storage.md`. I figure we can use that location to store similar tutorials for the time being. I have plans to write quite a few more, so we'll need some spot for them until the new learn site is launched.

##### Component Name

docs

### CODE REVIEW

File: docs/getting-started.md [hunk 1]
```diff
@@ -0,0 +1,237 @@
...
+> If you haven't installed Netdata yet, visit the [installation instructions](../packaging/installer) for details,
+> including our one-liner script, which automatically installs Netdata on almost all Linux distributions.
+
+## Access the dashboard
+
+Open up your web browser of choice. If you installed Netdata on the same system you're using to open your browser,
+navigate to `http://localhost:19999/`. If you installed Netdata on a remote system, navigate to
+`http://SYSTEM-IP:19999/`, replacing `SYSTEM-IP` with the IP address of that system.
+
+Hit `Enter`. Welcome to Netdata!
```
[the remark this answers is not in the document]
Comment (joelhans) [reply docs/getting-started.md]: I tend to be more verbose than necessary to be on the safe side, but this is good feedback. I can save some of this super granular stuff for the tutorial I'm working on!

[the remark this answers is not in the document]
Comment (joelhans) [reply docs/getting-started.md]: Tried to strike a slightly different tone in the most recent commit. Let me know what you think when you can.

File: docs/getting-started.md [hunk 2]
```diff
@@ -0,0 +1,237 @@
...
+## Collect data from more sources
+
+When Netdata **starts**, it auto-detects data sources, such as database servers, web servers, and more. To auto-detect
+and collect metrics from a service or application you just installed, you need to [restart
+Netdata](#start-stop-and-restart-netdata).
+
+> There is one exception: When Netdata is running on the host (as in not in a container itself), it will always
+> auto-detect containers and VMs.
+
+However, auto-detection only works if you configured the source correctly. If Netdata isn't collecting metrics after a
```
[the remark this answers is not in the document]
Comment (joelhans) [reply docs/getting-started.md]: Good point! Tweaked this sentence.

File: docs/getting-started.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (cakrit) [reply docs/getting-started.md]: I believe he means that all we need to write here is "Open up your web browser of choice and navigate to http://[YOUR HOST]:19999".

**Comment by cakrit** [comment]
@cosmix can you plz review again?

**Pull request merged**